javascript/javascript2/week1/warmUp.js

```diff
@@ -0,0 +1,44 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function getShortestWord(danishWords) {
+    console.log(
+        danishWords.reduce(function(a, b) {
+          return a.length <= b.length ? a : b;
+        })
+      )
+};
+getShortestWord(danishWords);
+
+
+const danishString = "Jegø harå enå blå bilæÆ";
+
+
+function findDanishLetters(danishString) {
+    const danishLetterØ = [];
+    const danishLetterÅ = [];
+    const danishLetterÆ = [];
+    danishString = danishString.split("");
+    for (i=0; i<danishString.length; i++) {
+        if (danishString[i] === "å") {
```

Comment:
would toLower work with danish letters? could you test it and see if it works?

Reply:
Yes, you're right. Thank you!